src/styles/blocks/card.scss

```diff
@@ -0,0 +1,113 @@
+.card {
+  height: 406px;
+  width: 198px;
+
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    margin: 32px auto 0;
+    height: 134px;
+    width: 160px;
+
+    background-image: url(../images/imac.jpeg);
+    background-size: 100% 100%;
+  }
+
+  &__title {
+    margin: 40px 16px 0;
+    font-family: Roboto, Arial, Helvetica, sans-serif;
```

Comment:
The font-family should be extracted to a variable for reusability and to ensure consistency across your stylesheets. This is mentioned in the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).